### PR TITLE
Add Cloud Custodian Annual report for 2021

### PR DESCRIPTION
**Pull request by castrojo**
Co-authored-by: Kapil Thangavelu <kapil@stacklet.io>
Co-authored-by: Liz Acosta <liz@stacklet.io>

### CODE REVIEW

File: reviews/2021-cloud-custodian-annual.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+## Goals for 2022
+
+- New enhancement process for feature development
+- GCP moving from beta to GA
+- ARM64 docker images
+- Extracting core from the AWS provider (have each cloud provider in its own package)
+- Kubernetes Admission Controller
+- Policy Enforcement against IaC definitions like Terraform (aka shift left)
+- Rigorous policy testing
```
Comment (kapilt) [inline reviews/2021-cloud-custodian-annual.md:79]: s/end user policy testing framework

File: reviews/2021-cloud-custodian-annual.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+More security details in: 
+[Joint Review](https://github.com/cncf/tag-security/blob/cf1f38ff803fb3f830209e16618221844a9dea68/assessments/projects/custodian/joint-review.md)
+[Self-assessment](https://github.com/cncf/tag-security/blob/ef2d51d54697f4037a4a697b405d270df2579620/assessments/projects/custodian/self-assessment.md)
+
+Note: Both documents pending merge, will update accordingly
+
+# Adoption
+
+[ADOPTERS.md](https://github.com/cloud-custodian/cloud-custodian/ADOPTERS.md)
```
Comment (resouer) [inline reviews/2021-cloud-custodian-annual.md:147]: 404

Comment (amye) [reply reviews/2021-cloud-custodian-annual.md:147]: @castrojo - where should this link to?

Comment (castrojo) [reply reviews/2021-cloud-custodian-annual.md]: Fixed, sorry for the late response, I was at a conference!

**Pull request merged**